service/src/testRunner.js

```diff
@@ -31,6 +31,28 @@ const runJest = async function(chromePath, ...args) {
     return result
 }
 
+const logResults = function(results, testVariables, retryCount) {
+    const newResult = new Object()
```

Comment:
I think object literal is usually preferred.  
```suggestion
    const newResult = {}
```